pkgs/applications/misc/yokadi/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, buildPythonApplication, dateutil,
+  sqlalchemy, setproctitle, icalendar, pycrypto }:
+
+buildPythonApplication rec {
+  pname = "yokadi-${version}";
```

Comment:
`pname` should not include version:
```
pname = "yokadi";
```

They are mixed in together in https://github.com/NixOS/nixpkgs/blob/4a2340ff6bd0474d9a3e933f28b8568c59019b82/pkgs/development/interpreters/python/mk-python-derivation.nix#L14.

Reply:
(Don't forget to fix `fetchurl` after this change.)